include/deal.II/matrix_free/matrix_free.templates.h

```diff
@@ -531,9 +372,10 @@ MatrixFree<dim, Number, VectorizedArrayType>::internal_reinit(
       n_components += dof_handler[no]->get_fe(0).n_base_elements();
     const unsigned int n_quad             = quad.size();
     unsigned int       n_fe_in_collection = 0;
-    for (unsigned int i = 0; i < n_components; ++i)
```

Comment:
I think this was not intended, wan it?

Reply:
Most definitely not :wink: - this tells something about the examples that have been tested (Navier-Stokes being the most complicated one with the first dof handler containing the maximal number of FEs in the collection).